tests/CLIFramework/LoggerTest.php

```diff
@@ -16,7 +16,7 @@ class LoggerTest extends TestCase
 {
     private $logger;
 
-    function setUp()
+    protected function setUp(): void
```

Comment:
According to the [PHPUnit doc](https://phpunit.readthedocs.io/en/9.5/fixtures.html), it should use the `protected function setUp(): void` and `protected function tearDown(): void` methods.